logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/user/ObserveSelfUserWithTeamUseCaseTest.kt

```diff
@@ -0,0 +1,79 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.kalium.logic.feature.user
+
+import app.cash.turbine.test
+import com.wire.kalium.logic.data.team.Team
+import com.wire.kalium.logic.data.user.SelfUser
+import com.wire.kalium.logic.data.user.UserRepository
+import com.wire.kalium.logic.framework.TestTeam
+import com.wire.kalium.logic.framework.TestUser
+import io.mockative.Mock
+import io.mockative.given
+import io.mockative.mock
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.runTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class ObserveSelfUserWithTeamUseCaseTest {
+
+    private val arrangement = Arrangement()
+
+    @Test
+    fun testIfValidDataIsPassedForward_0() = runTest {
```

Comment:
What's the difference between `testIfValidDataIsPassedForward_0` and `testIfValidDataIsPassedForward_1`?

The name could be clearer and explain what each test is actually testing.

Also, no need to add `testIf` before each test.

Reply:
well, the UseCase is quit simple: just get data from repository and pass it forward, so I wasn't even sure about the names for tests :) 
But I'll try to make something up